README.md

```diff
@@ -27,13 +27,25 @@ The currently planned/on-going integration work includes:
 
 Stay tuned!
 
-### Build instructions
+## Build instructions
 
-The basic commands to clone, build, and install brunsli are:
+### Linux
 
-    $ git clone https://github.com/google/brunsli.git
-    $ cd brunsli
-    $ git submodule update --init --recursive
-    $ cmake ./
-    $ make -j
-    $ make -j install
+The basic commands to clone, build, and install brunsli on Linux are:
+
+    git clone https://github.com/google/brunsli.git
+    cd brunsli
+    git submodule update --init --recursive
+    cmake ./
```

Comment:
it is better to build in a separate dir e.g. ./out or ./build

Reply:
Agreed, I just copied them as they existed